test/autorest/lrogroup/lros_test.go

```diff
@@ -236,6 +236,40 @@ func TestLROBeginDeleteProvisioning202Deletingcanceled200(t *testing.T) {
 	}
 }
 
+func TestLROBeginPatch201RetryWithAsyncHeader(t *testing.T) {
+	op := newLROSClient()
+	resp, err := op.BeginPatch201RetryWithAsyncHeader(context.Background(), nil)
+	require.NoError(t, err)
+	res, err := resp.PollUntilDone(context.Background(), time.Second)
+	require.NoError(t, err)
+	if r := cmp.Diff(res.Product, Product{
+		ID:   to.StringPtr("/lro/patch/201/retry/onlyAsyncHeader"),
+		Name: to.StringPtr("foo"),
+		Properties: &ProductProperties{
+			ProvisioningState: to.StringPtr("Succeeded"),
+		},
+	}); r != "" {
```

Comment:
Do you think we should use something like `require.Equal()` here instead?  I know we have this lingering pattern in the other tests, I just haven't been able to update them yet (it might be a difficult thing to do).  Minor issue.

Reply:
May as well. I wrote this before you rolled out `require` and only did the easy transforms after my rebase. The old pattern is regular enough that replacing it wasn't so bad (I did the rest of the file).